node/subsystem-types/src/messages.rs

```diff
@@ -219,6 +224,13 @@ pub enum CollatorProtocolMessage {
 	///
 	/// The hash is the relay parent.
 	Seconded(Hash, SignedFullStatement),
+	/// The candidate received enough validity votes from the backing group.
+	Backed {
```

Comment:
This doesn't look like it is actually sent

Reply:
It's sent from backing because we need new para head
https://github.com/paritytech/polkadot/blob/7b9cb2c832f2b5c15d21d511d5103b63757ed6bd/node/core/backing/src/lib.rs#L1568-L1574

Reply:
I see. That is also available in the prospective parachains subsystem via node storage